recipe/meta.yaml

```diff
@@ -11,28 +11,31 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install --no-deps . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  skip: true # [py<39]
 
 requirements:
   host:
     - pip
-    - python {{ python_min }}
+    - python
     - hatchling >=1.8.0
     - hatch-vcs
   run:
-    - python >={{ python_min }}
+    - python
     - typing_extensions  # not actually required at runtime
     - wrapt  # provides 'proxy' extra
 
+# typing_extensions is not actually required at runtime, but is required for
```

Comment:
Should we just add it as a test dependency instead then?

Reply:
That makes sense. No problem!